client/src/components/Footer/Footer.js

```diff
@@ -85,6 +85,14 @@ function Footer() {
               </span>
               and open-sourced
             </p>
+
+            <p>
+              The website was designed by{" "}
+              <a className="credit" href="https://github.com/EricLiclair">
+                {" "}
+                @EricLiclair.
+              </a>
+            </p>

```

Comment:
@ashwani3011, thanks for your PR, can you change the text to - `This` website `is` designed by @EricLiclair.